diff --git a/node_modules/@types/node-media-server/index.d.ts b/node_modules/@types/node-media-server/index.d.ts
index 47b372d..11d2ad5 100755
--- a/node_modules/@types/node-media-server/index.d.ts
+++ b/node_modules/@types/node-media-server/index.d.ts
@@ -3,7 +3,7 @@
 // Definitions by: Thomas Seberechts <https://github.com/Tseberechts>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-interface Config {
+export interface Config {
     logType?: number;
     rtmp?: RtmpConfig;
     http?: HttpConfig;
@@ -14,7 +14,7 @@ interface Config {
     auth?: AuthConfig;
 }
 
-interface RtmpConfig {
+export interface RtmpConfig {
     port?: number;
     ssl?: SslConfig;
     chunk_size?: number;
@@ -23,40 +23,40 @@ interface RtmpConfig {
     ping_timeout?: number;
 }
 
-interface SslConfig {
+export interface SslConfig {
     key: string;
     cert: string;
     port?: number;
 }
 
-interface HttpConfig {
+export interface HttpConfig {
     mediaroot: string;
     port?: number;
     allow_origin?: string;
 }
 
-interface AuthConfig {
+export interface AuthConfig {
     play?: boolean;
     publish?: boolean;
     secret?: string;
 }
 
-interface TransConfig {
+export interface TransConfig {
     ffmpeg: string;
     tasks: [TransTaskConfig];
 }
 
-interface RelayConfig {
+export interface RelayConfig {
     tasks: [RelayTaskConfig];
     ffmpeg: string;
 }
 
-interface FissionConfig {
+export interface FissionConfig {
     ffmpeg: string;
     tasks: [FissionTaskConfig];
 }
 
-interface TransTaskConfig {
+export interface TransTaskConfig {
     app: string;
     hls?: boolean;
     hlsFlags?: string;
@@ -72,7 +72,7 @@ interface TransTaskConfig {
     mp4Flags?: string;
 }
 
-interface RelayTaskConfig {
+export interface RelayTaskConfig {
     app: string;
     name?: string;
     mode: string;
@@ -80,12 +80,12 @@ interface RelayTaskConfig {
     rtsp_transport?: string;
 }
 
-interface FissionTaskConfig {
+export interface FissionTaskConfig {
     rule: string;
     model: [FissionTaskModel];
 }
 
-interface FissionTaskModel {
+export interface FissionTaskModel {
     ab: string;
     vb: string;
     vs: string;
